example/README.md

```diff
@@ -231,34 +233,40 @@ Workflow tests are located in the `/spec/<workflow-name>` folder. Any tests and
+Ideally the test can run in parallel with other tests and should be put in the `parallel` folder. If it cannot be, it should go in the `spec/serial` folder. Only if the test should be run outside of the test suite should it go in the `standalone` folder.
 
 ## Fake data server
 
 A fake server is required for tests testing FTP/HTTP/HTTPS discover and downloads. The fake server should be set up once per account.
 
-The Cloudformation template for the fake data server is in `fake-server.yml`. To setup the fake server run:
+The Cloudformation template for the fake data server is in `fake-provider-cf.yml`. To setup the fake server, you can use the AWS CLI or AWS Console.
+
+If you want to use the AWS CLI, run:
 
 ```bash
-aws cloudformation deploy --template-file fake-server.yml --stack-name <stack-name> --parameter-overrides VpcId=<vpc-XXXXX> SubnetId=<subnet-XXXXXX> AZone=<az-zone> Ngap=true --capabilities CAPABILITY_NAMED_IAM
+aws cloudformation deploy --template-file fake-provider-cf.yml --stack-name <stack-name> --parameter-overrides VpcId=<vpc-XXXXX> Subnet=<subnet-XXXXXX> PermissionsBoundary=<permissions-boundary> NGAPProtAppInstanceMinimalPolicyName=<policy-name> LatestAmiId=<ami-id> FtpPassword=<ftp-password> Bucket=<bucket-name> Prefix=<prefix> --capabilities CAPABILITY_NAMED_IAM
 ```
 
 with the following parameters:
 
 - stack-name - Stack name for the fake server
 - VpcId - VPC ID
-- SubnetId - Subnet ID
-- AZone - Availability zone, needs to match the Subnet ID's availability zone
-- Ngap - `true` if in an NASA NGAP environment, will add the NGAP permission boundary to the IAM role created
+- Subnet - Subnet ID
+- PermissionsBoundary - A permissions boundary from NGAP. 
+- NGAPProtAppInstanceMinimalPolicyName - Will be included in the list of Amazon Resource Names (ARNs) of the IAM managed policies we want to attach to the user.
+- LatestAmiId - The most recent Amazon Machine Image ID
+- FtpPassword -
+- Bucket - 
+- Prefix - Any string, generally a DAAC's name
 
-In the outputs section of your Cloudformation deployment in the AWS console, you can find the IP address of the fake server created. To use this fake server with the tests, update the provider configurations in `example/data/providers` to use this host address.
+Alternatively, you can use the AWS Console. Navigate to `CloudFormation > Create Stack with new resources > Create template in designer` and paste the contents of `fake-provider-cf.yml` into the template box. Be sure to choose `YAML` as the template language.
 
 ### Update data bucket
 
-By default, the data location is the `cumulus-data-shared` S3 bucket. To use a different bucket for test data, update `fake-server.yml` with the alternative bucket and re-deploy the fake data server.
+By default, the data location is the `cumulus-data-shared` S3 bucket. To use a different bucket for test data, update `fake-provider-cf.yml` with the alternative bucket and re-deploy the fake data server.
```

Comment:
I'm not sure this is relevant anymore either. There is no `cumulus-data-shared` bucket on our account. I think all the test data is manually added to the EC2 instance by the startup script in `fake-provider-cf.yml`, so I believe this section can just go away